components/menu/menu-item-mixin.js

```diff
@@ -8,6 +8,7 @@ export const MenuItemMixin = superclass => class extends superclass {
 			hidden: { type: Boolean, reflect: true },
 			last: { type: String, reflect: true }, // set by d2l-menu
 			role: { type: String, reflect: true },
+			tabindex: { type: String, reflect: true },
```

Comment:
Hmmm... this doesn't conflict with native `tabindex` property/attribute that exists on all HTML elements?